client/components/productCard.js

```diff
@@ -1,16 +1,14 @@
 import React from 'react'
 import {Link} from 'react-router-dom'
-import {AddToCart} from './index'
 
 const ProductCard = props => {
   return (
-    <li key={props.product.id}>
+    <li>
       <Link to={`./products/${props.product.id}`}>
```

Comment:
Can you delete the . in all the <Link> paths. The ./ is used for referencing directories and / is used for url paths.